plugins/inputs/wireless/wireless.go

```diff
@@ -14,8 +14,9 @@ type Wireless struct {
 var sampleConfig = `
   ## Sets 'proc' directory path
   ## If not specified, then default is /proc
+  ## Ignored on macOS/darwin
   # host_proc = "/proc"
-`
+  `
```

Comment:
Can you please revert this (whitespace) change.

Reply:
Pretty sure I was asked to _add_ that whitespace earlier. Is a single space really worth all of this? https://github.com/influxdata/telegraf/pull/10661#discussion_r808797157

Reply:
Pretty sure I asked to revert it (aka remove). Don’t know why you resolved that conversation without actually resolving?

Reply:
Oh, I see. You added a space instead of removing it.

@davidgs yes, these kind of reviews are needed to maintain clean, working code and similar behaviour across the product.

Reply:
You said: 
> Please revert the added space at the end and fix indentation difference (should be 2 spaces).

Perhaps I misunderstood that you actually wanted. I absolutely understand the need for thorough code reviews. I also understand the need to maintain formatting standards, etc. but we have now gone 4 rounds over a _single space_ which would not, in any way, affect the behavior of the product.

Reply:
That second part of my sentence was of course about line 17, which is fixed in the meantime. No problem, it was a misunderstanding, that can happen.

This specific part will impact the config file generation to be not consistent anymore. So yes, still important.

Reply:
@davidgs I for my part want you to remove the two spaces you introduced in line 19 (the closing quote line).

Reply:
Done.